HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -4182,7 +4182,8 @@ def self.add_variable_speed_power_ems_program(runner, model, air_loop_unitary, c
         program.addLine("  If #{clg_tot_sensor.name} > 0")
         program.addLine("    Set #{cap_multiplier} = #{clg_sens_sensor.name} / #{clg_tot_sensor.name}")
         program.addLine('  Else')
-        program.addLine("    Set #{cap_multiplier} = 0.0")
```

Comment:
The spikes happened when the point is the first hour in a cooling cycle, and the SHR was set to 0.0 here, which confused the EMS to execute the max power cutoff (thought that the load is greater than available capacity)

Reply:
Pulled your commit into a separate PR so we can see the change by itself: https://github.com/NREL/OpenStudio-HPXML/pull/1909